src/components/policy/policy_external/src/sql_pt_queries.cc

```diff
@@ -945,7 +949,7 @@ const std::string kInsertApplicationFull =
     " `is_predata`, `memory_kb`, `heart_beat_timeout_ms`, "
     " `certificate`, `hybrid_app_preference_value`, `endpoint`, `enabled`, "
     " `auth_token`, `cloud_transport_type`, `icon_url`, "
-    "`allow_unknown_rpc_passthrough`) "
+    "`allow_unknown_rpc_passthrough`, `encryption_required`) "
     "  VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
```

Comment:
missing  extra "?" because of the new parameter added

Reply:
fixed in 45f0888b531adde8ac91715a3c526c75c7c23ba3